gym/lib/gym/options.rb

```diff
@@ -129,6 +133,13 @@ def self.plain_options
                                      env_name: "GYM_SKIP_BUILD_ARCHIVE",
                                      description: "Export ipa from previously built xarchive. Uses archive_path as source",
                                      is_string: false,
+                                     type: Boolean,
+                                     optional: true),
+        FastlaneCore::ConfigItem.new(key: :skip_archive,
```

Comment:
new feature?

Reply:
Yeah, we're using gym to build the `FastlaneRunner` binary after Swift API generation, but gym really wants to create a fully ITC capable package.